src/main/java/it/pagopa/pn/user/attributes/middleware/wsclient/PnExternalRegistryClient.java

```diff
@@ -38,4 +50,15 @@ public Mono<String> findPrivacyNoticeVersion(String consentType, String portalTy
                 .map(Object::toString);
     }
 
+    @Cacheable(value = "aooSenderIdCache")
+    public Mono<String> getRootSenderId(String id){
```

Comment:
il @cacheable con il Mono quando ci avevo guardato io (inizio anno scorso) avevo letto che non funzionava come ci si aspettava (nel senso che cachava il "mono" e non il "string" che ci si aspetta)
Hai fatto un check che effettivamente sta cosa funzioni come ci si aspetti (ovvero che la cache non faccia invocazioni verso ext-registry se il valore è in cache)?